spec/controllers/application_controller_spec.rb

```diff
@@ -25,7 +31,7 @@ def index
 
     context "user access token is not authorized" do
       before do
-        User.any_instance.stub(:authorized_access_token?).and_return(false)
+        allow_any_instance_of(User).to receive(:authorized_access_token?).and_return(false)
```

Comment:
This doesn't change any behavior but gets rid of the deprecation warning for the old `any_instance` syntax.